packages/transmute-framework/contracts/EventStore.sol

```diff
@@ -1,39 +1,16 @@
 pragma solidity ^0.4.19;
 
-import "./EventStoreLib.sol";
-
 contract EventStore {
+    uint public count;
 
-    EventStoreLib.TransmuteStorage store;
-
-    address public owner;
-
-    function () public payable { revert(); }
- 
-    constructor() public {
-        owner = tx.origin;
-    }
-
-    function count() public view 
-    returns (uint) {
-        return store.events.length;
-    }
-
-    function write(bytes32 key, bytes32 value) public {
-        EventStoreLib.write(
-            store,
-            key,
-            value
-        );
-    }
-    
-    function read(uint index) public view
-    returns (uint, address, bytes32, bytes32 ) {
-        return EventStoreLib.read(store, index);
-    }
+    event TransmuteEvent(
+        uint index,
+        address indexed sender,
```

Comment:
Is this information included in the transaction logs, I don't think we need it here.

Reply:
This would be useful if we want to efficiently lookup events by sent addresses who sent them. Not sure if we need that